assets/scss/shortcodes/cards-pane.scss

```diff
@@ -24,7 +24,7 @@
 
     pre {
       margin: 0;
-      padding: 0 1rem 1rem 1rem;
+      padding: 1rem;
```

Comment:
AFAICT, you can drop this because it's the "global" setting under `.td-content .highlight pre` (and `.td-content pre`:

```suggestion
```

Reply:
Line removed as requested.